README.md

```diff
@@ -83,6 +83,20 @@ if variant.enabled {
 }
 ```
 
+### Available options
+
+The Unleash SDK takes the following options:
+
+| option            | required | default                   | description                                                                                                                                      |
+|-------------------|----------|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| unleashUrl        | yes | n/a                       | The Unleash Proxy URL to connect to. E.g.: `https://examples.com/proxy`                                                                         |
```

Comment:
Should we say here: Proxy/Edge/Frontend API?

Reply:
We need to consistently add it across all SDKs